### DISCUSSION TITLE
the NSFW filter issues ,bad anatomy - change in the prompt understanding -not correct for generation

### DISCUSSION THREAD

**Discussion by HAMSAILY** [Acly/krita-ai-diffusion#1063] [General]
After i chose the check for nsfw ,i put it basic ,or strict ,its give me bad in anatomy or not perfect in prompting until if I off or disable still not disable ,I close the Krita  restart the plugin ai still the tool "nsfw" working

**Comment by Acly** [comment]
NSFW filter does not affect generation at all.

**Comment by monkyboom** [comment]
it does it makes life boring so it does,
